### PR TITLE
Remove the OVN NetworkManager Event Catcher

### PR DESCRIPTION
**Pull request by agrare**
RHV with the OVN service enabled does not have ceilometer, stf, or amqp so there is nothing for the event catcher to connect to.

Schema migration: https://github.com/ManageIQ/manageiq-schema/pull/684

https://github.com/ManageIQ/manageiq-providers-ovirt/issues/631

### CODE REVIEW

**Comment by Fryguy** [comment]
@agrare Not sure why this is failing as the error seems unrelated.

**Comment by agrare** [comment]
Oh it is because there was a `{:class_name => "ManageIQ::Providers::Ovirt::NetworkManager", :method_name => "stop_event_monitor"}` on the queue due to the authentication change, and now that isn't there anymore.  Will update the count

**Comment by agrare** [comment]
@Fryguy fixed

File: config/settings.yml [hunk 1]
```diff
@@ -64,17 +64,6 @@
     :event_catcher:
       :event_catcher_ovirt:
         :poll: 15.seconds
-      :event_catcher_ovirt_network:
```
Comment (Fryguy) [inline config/settings.yml:67]: Do we need a data migration to remove these from the settings changes table if they exist?

Comment (agrare) [reply config/settings.yml:67]: :+1: yeah that is true I'll update the schema PR with this

**Comment by agrare** [comment]
Will need to create a petrosian direct PR similar to https://github.com/ManageIQ/manageiq-providers-red_hat_virtualization/pull/25

**Pull request merged**